<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="databaseEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="loadDatabaseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImagesButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load Images</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imagesEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="chooseDescriptorlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose Descriptor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="sift">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIFT</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="siftlbpseparate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIFT and LBP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="siftorthogonallbp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIFT and OC-LBP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="siftlbphue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIFT, LBP, HUE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="siftorthogonallbphue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIFT, OC-LBP, HUE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HOG</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hoglbp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HOG and LBP</string>
    </property>
   </widget>
   <widget class="QLabel" name="siftWeightLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIFT/HOG Weight</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbpWeightLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OC-LBP/LBP Weight</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hueWeightLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HUE Weight</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="siftWeightEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lbpWeightEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="hueWeightEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="returnImagesLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number of images</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="returnImagesEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="prepareButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prepare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="queryButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Query</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="dictionarySizeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dictionary size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="dictionarySizeEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>461</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sift</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>siftlbpseparate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>siftorthogonallbp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>siftlbphue</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>siftorthogonallbphue</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hog</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hoglbp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prepareButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>prepareBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queryButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>queryBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>testButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>testBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>radioBtnSelected()</slot>
  <slot>prepareBtnSelected()</slot>
  <slot>queryBtnSelected()</slot>
  <slot>testBtnSelected()</slot>
 </slots>
</ui>
